File: app/src/main/java/spave/levan/waterever/model/Tag.java
package spave.levan.waterever.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * 佛祖保佑 永无BUG
 *
 * @author WangZhiYao
 * @date 2018/9/16
 */
public class Tag extends RealmObject {

    @PrimaryKey
    private String tag;

    public String getTag() {
        return tag;
    }

    public void setTag(String tag) {
        this.tag = tag;
    }
}

File: app/src/main/kotlin/me/zhiyao/waterever/data/db/model/Tag.kt
package me.zhiyao.waterever.data.db.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 *
 * @author WangZhiYao
 * @date 2020/8/10
 */
@Entity(tableName = "tags")
data class Tag(
    @ColumnInfo(name = "tag_name")
    val name: String,
    @ColumnInfo(name = "create_time")
    val createTime: Long
) {
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "tag_id")
    var id: Long = 0
}